.mergify.yml

```diff
@@ -67,25 +64,6 @@ queue_rules:
               - "status-success=ci/centos/mini-e2e/k8s-1.28"
               - "status-success=ci/centos/upgrade-tests-cephfs"
               - "status-success=ci/centos/upgrade-tests-rbd"
-          - and:
```

Comment:
Let's keep this ? 
We'll need to re add when k8s test versions differs between devel and release branches?

Reply:
I think we can do that when there is a need. Currently things (rules to queue, rules to merge) are not in-sync, this PR corrects that too.

Reply:
@nixpanic we need to readd everything in that case,. keep as its and remove 1.26 from devel for sure and if we are not planning to run that on release-3.11 we can drop it from above section. lets not remove the entire section for devel.

Reply:
Ok! I've added the rule for the devel branch in that case, so that everything is the same everywhere.